### ISSUE TITLE
Test Kitchen fails due to "missing 'VERSION' file"

### ISSUE THREAD

**Issue by brucewillke** [litaio/chef-lita#16]
I've clone this repo, because I was experiencing the same issue pulling in Lita from Berkshelf

```================================================================================
Error Resolving Cookbooks for Run List:
================================================================================

       Unknown Server Error:
       ---------------------

       The server had a fatal error attempting to load the node data.

       Server Response:
       ----------------
       Exception raised!  #<Errno::ENOENT: No such file or directory @ rb_sysopen - /tmp/kitchen/cookbooks/lita/VERSION>

       /tmp/kitchen/cookbooks/lita/metadata.rb:7:in `read'
       /tmp/kitchen/cookbooks/lita/metadata.rb:7:in `from_file'
       /opt/chef/embedded/apps/chef/lib/chef/mixin/from_file.rb:30:in `instance_eval'
       /opt/chef/embedded/apps/chef/lib/chef/mixin/from_file.rb:30:in `from_file'

       /opt/chef/embedded/apps/chef/lib/chef/cookbook/cookbook_version_loader.rb:253:in `apply_ruby_metadata'
       /opt/chef/embedded/apps/chef/lib/chef/cookbook/cookbook_version_loader.rb:161:in `block in metadata'
       /opt/chef/embedded/apps/chef/lib/chef/cookbook/cookbook_version_loader.rb:158:in `each'
       /opt/chef/embedded/apps/chef/lib/chef/cookbook/cookbook_version_loader.rb:158:in `metadata'
       /opt/chef/embedded/apps/chef/lib/chef/cookbook/cookbook_version_loader.rb:76:in `load'
       /opt/chef/embedded/apps/chef/lib/chef/chef_fs/file_system/chef_repository_file_system_cookbook_dir.rb:46:in `chef_object'

       /opt/chef/embedded/apps/chef/lib/chef/chef_fs/chef_fs_data_store.rb:150:in `block in get'
       /opt/chef/embedded/apps/chef/lib/chef/chef_fs/chef_fs_data_store.rb:439:in `with_entry'
       /opt/chef/embedded/apps/chef/lib/chef/chef_fs/chef_fs_data_store.rb:145:in `get'
       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-zero-3.2.1/lib/chef_zero/data_store/v1_to_v2_adapter.rb:47:in `block in get'
       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-zero-3.2.1/lib/chef_zero/data_store/v1_to_v2_adapter.rb:113:in `fix_exceptions'

       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-zero-3.2.1/lib/chef_zero/data_store/v1_to_v2_adapter.rb:40:in `get'
       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-zero-3.2.1/lib/chef_zero/rest_base.rb:52:in `get_data'
       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-zero-3.2.1/lib/chef_zero/endpoints/environment_cookbook_versions_endpoint.rb:77:in `block in depsolve'
       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-zero-3.2.1/lib/chef_zero/endpoints/environment_cookbook_versions_endpoint.rb:71:in `each'
       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-zero-3.2.1/lib/chef_zero/endpoints/environment_cookbook_versions_endpoint.rb:71:in `depsolve'
       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-zero-3.2.1/lib/chef_zero/endpoints/environment_cookbook_versions_endpoint.rb:37:in `post'

       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-zero-3.2.1/lib/chef_zero/rest_base.rb:30:in `call'
       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-zero-3.2.1/lib/chef_zero/rest_router.rb:24:in `call'
       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-zero-3.2.1/lib/chef_zero/server.rb:554:in `block in app'
       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/rack-1.5.2/lib/rack/handler/webrick.rb:60:in `call'
       /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/rack-1.5.2/lib/rack/handler/webrick.rb:60:in `service'
       /opt/chef/embedded/lib/ruby/2.1.0/webrick/httpserver.rb:138:in `service'

       /opt/chef/embedded/lib/ruby/2.1.0/webrick/httpserver.rb:94:in `run'
       /opt/chef/embedded/lib/ruby/2.1.0/webrick/server.rb:295:in `block in start_thread'



Running handlers:
[2015-01-08T15:22:33+00:00] ERROR: Running exception handlers
       Running handlers complete
       [2015-01-08T15:22:33+00:00] ERROR: Exception handlers complete
[2015-01-08T15:22:33+00:00] FATAL: Stacktrace dumped to /tmp/kitchen/cache/chef-stacktrace.out
Chef Client failed. 0 resources updated in 82.119693841 seconds
[2015-01-08T15:22:33+00:00] ERROR: 500 "Internal Server Error "
[2015-01-08T15:22:35+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)```

**Comment by harlanbarnes** [comment]
Hmm. Not sure. I just cloned a new copy into a temp directory and did `kitchen congverge` and it ran successfully. I checked the Cookbook download and the VERSION file is there in the tar ball.

Here are the ideas I have ...

1. Maybe this is some issue with Ruby 2.1.0. I'm still using Chef omnibus 11.16.4 so I'm still Ruby 1.9.1.
2. Maybe this version of Chef Zero (or the Chef client maybe?) is more strict about what it uploads to the server?

Actually, who cares? :-)

We can move the version into metadata.rb. This is a pattern I've used before that makes it easier to bump versions through CI or such. But I'm not using CI here. Let me fix this up when I get done with work today and I'll make a release tonight.
